collada/lineset.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, sources, material, index, xmlnode=None):
         """
 
         if len(sources) == 0: raise DaeIncompleteError('A line set needs at least one input for vertex positions')
-        if not 'VERTEX' in sources: raise DaeIncompleteError('Line set requires vertex input')
+        if len(sources.get('VERTEX', [])) == 0: raise DaeIncompleteError('Line set requires vertex input')
```

Comment:
I think you can simplify this with:
`if not sources.get('VERTEX'):`

Reply:
Totally right. Don't know how I missed that. :)